app/models/source.rb

```diff
@@ -26,5 +26,4 @@ class Source < ApplicationRecord
   has_many :tasks
 
   ALLOWED_REFRESH_STATUS_VALUES = ["deployed", "quota_limited"].freeze
```

Comment:
@hsong-rh Can you also delete this since it's not needed anymore. Since it was used in the earlier design to figure out if a pod has been deployed for doing the inventory collection.

Reply:
Removed